.travis.yml

```diff
@@ -33,10 +33,7 @@ addons:
       - libclang-8-dev
 
 install:
-  - export TAG=v8.0.1-3
-  - export TARBALL=SPIRV-LLVM-Translator-${TAG}-linux-${BUILD_TYPE}.zip
-  - wget https://github.com/KhronosGroup/SPIRV-LLVM-Translator/releases/download/${TAG}/${TARBALL} -O /tmp/${TARBALL}
-  - unzip /tmp/${TARBALL} -d spirv-llvm-translator
+  - git clone https://github.com/KhronosGroup/SPIRV-LLVM-Translator.git -b llvm_release_80 spirv-llvm-translator
```

Comment:
I'm not sure that this would be enough

Basically, we are not triggering build of the translator here.

If I understand correctly, we have to git clone the whole llvm, put both the translator and opencl-clang in there, and only then perform in-tree build
Or, we need to build the translator first as separate cmake invocation and then supply it into opencl-clang build

Reply:
Thank you. I've updated it.

Reply:
Indeed, you don't have to build llvm. It will take a lot of time, probably travis job will time out.
I think we should build translator as a separate step in cmake. This build should use llvm binaries from deb packages as it is done now.

Reply:
You're right. Updated.

Reply:
@AlexeySachkov , @AlexeySotkin , could you help review when you're available? Thank you.